### ISSUE TITLE
Don't use aria-label

### ISSUE THREAD

**Issue by MatzeKitt** [deworg/webring#2]
The current snippet uses `aria-label` for some links. As I just read today, this is not recommended (and even violates WCAG SC [2.5.3 Label in Name](https://www.w3.org/WAI/WCAG22/Understanding/label-in-name.html)), as per: https://adrianroselli.com/2026/01/barriers-from-links-with-aria.html

Can we replace this with something better? Since we’re outside of WordPress context, `screen-reader-text` class cannot be used. Maybe inline CSS, which does the same?

**Comment by 2ndkauboy** [comment]
Since this snippet should be usable on any website, not only WordPress websites, having additional CSS can be hard to achieve. Also implementing a `.screen-reader-text`  inline [is quite long](https://www.a11yproject.com/posts/how-to-hide-content/).

How about using "Prev" and "Next" instead of an arrow/caret to avoid having to use an `aria-label` in the first place? We could also use a `span` inside the link with the `aria-label` which should work.

**Comment by MatzeKitt** [comment]
> How about using "Prev" and "Next" instead of an arrow/caret to avoid having to use an aria-label in the first place?

Sounds good to me!
